<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewer_MainWindow</class>
 <widget class="QMainWindow" name="viewer_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>813</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTabWidget" name="configuration_tabWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>150</height>
            </size>
           </property>
           <property name="tabPosition">
            <enum>QTabWidget::West</enum>
           </property>
           <property name="currentIndex">
            <number>3</number>
           </property>
           <property name="movable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="atombond_tab">
            <attribute name="title">
             <string>Atom/ Bond</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="1">
              <widget class="QGroupBox" name="rendermodel_groupBox">
               <property name="title">
                <string>Atom/Bond Model</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="4" column="0">
                 <widget class="QRadioButton" name="ssf_model_radiobutton">
                  <property name="text">
                   <string>Small Atoms</string>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QRadioButton" name="sf_model_radiobutton">
                  <property name="text">
                   <string>Space Fill</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QRadioButton" name="bs_model_radioButton">
                  <property name="text">
                   <string>Ball Stick</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QGroupBox" name="mol_rendermode_groupBox">
               <property name="title">
                <string>Atom Selection</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_5">
                <item row="0" column="0">
                 <widget class="QRadioButton" name="full_mol_rendermode_radioButton">
                  <property name="text">
                   <string>All</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QRadioButton" name="bb_mol_rendermode_radioButton">
                  <property name="text">
                   <string>Backbone Only</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QRadioButton" name="pocatoms_rendermode_radioButton">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="text">
                   <string>Pocket Atoms</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QRadioButton" name="none_rendermode_radioButton">
                  <property name="text">
                   <string>None</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="reload_pushButton">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../resources/proteinvis.qrc">
                 <normaloff>:/images/reload2.png</normaloff>:/images/reload2.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="2">
              <widget class="QTableView" name="protein_grouping_tableView">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectItems</enum>
               </property>
              </widget>
             </item>
             <item row="3" column="0" colspan="2">
              <widget class="QtGlMaterialEditor" name="atombond_materialEditor">
               <property name="title">
                <string>Material</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QComboBox" name="protein_grouping_comboBox">
               <property name="toolTip">
                <string>atom group by options</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="surface_tab">
            <attribute name="title">
             <string>Surface</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_10">
             <item row="0" column="0">
              <widget class="QGroupBox" name="showhide_surface_groupBox">
               <property name="title">
                <string>Show/Hide</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_7">
                <item row="0" column="0">
                 <widget class="QRadioButton" name="hide_surface_radioButton">
                  <property name="text">
                   <string>Hide Surface</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QRadioButton" name="show_surface_radioButton">
                  <property name="text">
                   <string>Show Surface</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QRadioButton" name="show_surface_wireframe_radioButton">
                  <property name="text">
                   <string>Show Wireframe</string>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QtGlMaterialEditor" name="surface_materialEditor">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="title">
                <string>Material</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_8"/>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="alpha_pocket_tab">
            <attribute name="title">
             <string>Alpha Shape/Pockets</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_11">
             <item row="1" column="0">
              <widget class="QGroupBox" name="pocket_groupBox">
               <property name="title">
                <string>Pockets</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <layout class="QFormLayout" name="formLayout">
                <property name="fieldGrowthPolicy">
                 <enum>QFormLayout::ExpandingFieldsGrow</enum>
                </property>
                <item row="0" column="0">
                 <widget class="QSpinBox" name="pocket_alpha_num_spinBox"/>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="pocket_alpha_num_label">
                  <property name="text">
                   <string>Alpha No.</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0" colspan="2">
                 <widget class="QRadioButton" name="show_all_pockets_radioButton">
                  <property name="text">
                   <string>Show All Pockets</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QSpinBox" name="pocket_num_spinBox">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLabel" name="pocket_num_label">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Pocket No.</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QGroupBox" name="groupBox">
               <property name="title">
                <string>Grow Radii</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="0" column="0">
                 <widget class="QDoubleSpinBox" name="add_radius_sf_model_doubleSpinBox">
                  <property name="maximum">
                   <double>999.990000000000009</double>
                  </property>
                  <property name="singleStep">
                   <double>0.100000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="add_radius_sf_model_label">
                  <property name="text">
                   <string>Constant</string>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QDoubleSpinBox" name="alpha_value_sf_model_doubleSpinBox">
                  <property name="maximum">
                   <double>999.990000000000009</double>
                  </property>
                  <property name="singleStep">
                   <double>0.100000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLabel" name="alpha_value_sf_model_label">
                  <property name="text">
                   <string>Alpha</string>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QGroupBox" name="alpha_complex_groupBox">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="title">
                <string>Alpha Complex</string>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <layout class="QGridLayout" name="gridLayout_13">
                <item row="0" column="0">
                 <widget class="QCheckBox" name="show_alpha_complex_tets_checkBox">
                  <property name="text">
                   <string>Show Tetras</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QCheckBox" name="show_alpha_complex_tris_checkBox">
                  <property name="text">
                   <string>Show Tris</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QCheckBox" name="show_alpha_complex_edges_checkBox">
                  <property name="text">
                   <string>Show Eges</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="secondaryTab">
            <attribute name="title">
             <string>Secondary</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QGroupBox" name="sec_renderModel_groupBox">
               <property name="title">
                <string>Type Selection</string>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <widget class="QRadioButton" name="complete_sec_radiobutton">
                  <property name="text">
                   <string>All</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QRadioButton" name="helix_sec_radiobutton">
                  <property name="text">
                   <string>Only Helices</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QRadioButton" name="sheets_sec_radiobutton">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="text">
                   <string>Only Sheets</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QRadioButton" name="none_rendermode_radioButton_2">
                  <property name="text">
                   <string>None</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QSpinBox" name="spinBox_2"/>
               </item>
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Sheet Number</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox"/>
               </item>
               <item>
                <widget class="QLabel" name="helixLabel">
                 <property name="text">
                  <string>Helix Number</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="current_protein_comboBox"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QtGlLightEditor" name="lightseditor">
           <property name="title">
            <string>Lights</string>
           </property>
           <layout class="QGridLayout" name="gridLayout"/>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="render_light_graphic_checkBox">
           <property name="text">
            <string>Render light graphics</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="glviewer_t" name="glviewer" native="true">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>400</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1035</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Protein"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_Surface"/>
    <addaction name="actionLoad_Alpha_Shape"/>
    <addaction name="actionLoad_Pockets"/>
    <addaction name="separator"/>
    <addaction name="actionClose_Protein"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Protein">
   <property name="text">
    <string>Open Protein</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionLoad_Alpha_Shape">
   <property name="text">
    <string>Load Alpha Shape</string>
   </property>
  </action>
  <action name="actionLoad_Pockets">
   <property name="text">
    <string>Load Pockets</string>
   </property>
  </action>
  <action name="actionLoad_Surface">
   <property name="text">
    <string>Load Surface</string>
   </property>
  </action>
  <action name="actionClose_Protein">
   <property name="text">
    <string>Close Protein</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>glviewer_t</class>
   <extends>QWidget</extends>
   <header>viewer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtGlMaterialEditor</class>
   <extends>QGroupBox</extends>
   <header>qtglmaterialeditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtGlLightEditor</class>
   <extends>QGroupBox</extends>
   <header location="global">qtgllighteditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/proteinvis.qrc"/>
 </resources>
 <connections/>
</ui>
